src/hooks/useSafeNotifications.ts

```diff
@@ -35,16 +36,18 @@ const useSafeNotifications = (): void => {
     if (!isOwner) return
     if (implementationVersionState !== ImplementationVersionState.OUTDATED) return
 
+    const isDismissed = local.getWithExpiry<boolean>(DISMISS_NOTIFICATION_KEY + '_' + chainId + ':' + safeAddress)
```

Comment:
This key is very specific and is also used in the middleware. I'd suggest creating a function for getting it.

I also don't think you should dispatch it if it's dismissed. It might be worth separating these two into independent effects.

Reply:
Nit: can you rename this to `isUpdatedDismissed` as this shouldn't relate to the unsupported notifications.

Reply:
> This key is very specific and is also used in the middleware. I'd suggest creating a function for getting it.
Good idea, I will extract it into a function.

> I also don't think you should dispatch it if it's dismissed.
It should be easy to change that however we decide. I don't think having the notification inside the center does any harm but we could also not dispatch it at all.